aminator/plugins/provisioner/apt.py

```diff
@@ -89,19 +81,37 @@ def _store_package_metadata(self):
 
     @staticmethod
     def dpkg_install(package):
-        return monitor_command(['dpkg', '-i', package])
+        dpkg_result = monitor_command(['dpkg', '-i', package])
+        if not dpkg_result.success:
+            log.debug('failure:{0.command} :{0.std_err}'.format(dpkg_ret.result))
```

Comment:
Rather than repeat these log.debug messages with the contents of monitor_command results, seemed to make sense to have it done inside the method instead of inside the caller so it's done consistently every time